pkg/mon/logger_output_iowriter.go

```diff
@@ -0,0 +1,38 @@
+package mon
+
+import (
+	"fmt"
+	"github.com/applike/gosoline/pkg/clock"
+	"io"
+	"os"
+)
+
+func newIowriterLoggerWithInterfaces(clock clock.Clock, formatter formatter, output *os.File, timestampFormt string) (Handler, error) {
```

Comment:
Like the abstraction here, maybe you could just pass a writer interface, you can pass different things than os.File?

Also: `newIoWriter` and `timestampFormat` :D

Reply:
And maybe make this one public, so others can use it too